website/contributing/versioning-policy.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Versioning Policy
+---
+
+This page describes the versioning policy we follow for the `react-native` package.
+
+We test each version of React Native thoroughly, both with manual and automated tests, to ensure the quality doesn't regress.
+
+The `stable` channel of React Native follows the 0.x.y release policy described below.
+
+React Native also offers a `nightly` release channel to encourage early feedback on experimental features.
+
+This page describes our approach to version numbers for `react-native` and packages under the `@react-native` scope.
+
+## Stable Release Versions
+
+React Native releases stable versions at a regular cadence.
+
+We follow the 0.x.y versioning schema as follows:
+
+- Breaking changes will be shipped in a new minor version, i.e. we increment the x number (e.g.: 0.78.0 to 0.79.0).
+- New features and APIs will also be shipped in a new minor version, i.e. we increment the x number (e.g.: 0.78.0 to 0.79.0).
+- Critical bug fixes will be shipped in a new patch version, i.e. we increment the y number (e.g.: 0.78.1 to 0.78.2).
+
+Stable releases are shipped regularly, with the latest tagged as `latest` on NPM.
+
+A series of releases under the same minor number is called a **minor serie** (e.g. 0.76.x is the minor series for 0.76.0, 0.76.1, 0.76.2, etc.).
+
+## Commitment to Stability
+
+In order to support users upgrading React Native versions, we’re committed to maintain the latest 3 minor series (e.g. 0.78.x, 0.77.x and 0.76.x when 0.78 is the latest release).
+
+For those releases we’ll be publishing regular updates and bug fixes. In the event of a serious security vulnerability or significant disruption, we’ll be releasing updates for minor series outside the latest 3.
```

Comment:
```suggestion
For those releases we’ll be publishing regular updates and bug fixes.
```

I don't think we can anymore. We don't have the infra to release 0.74 and lower, for example.